buf.yaml

```diff
@@ -1,11 +1,11 @@
 version: v2
 modules:
   - path: proto
+    excludes:
+      - proto/google/protobuf
```

Comment:
No need to actually build these files in practice, as the generated message types are distributed as part of our python runtime anyways.